### ISSUE TITLE
The enemy's characters fall into the water

### ISSUE THREAD

**Issue by Shahin3d** [Dark-Basic-Software-Limited/GameGuruRepo#144]
The enemy's characters fall into the water !!!
When the **enemy characters** come looking for you - if the **water** is between you and them - the enemy characters fall into the water.

![en1](https://user-images.githubusercontent.com/36647058/38113625-c8fcea36-33ba-11e8-9532-d26541ca7c6e.png)
![en2](https://user-images.githubusercontent.com/36647058/38113626-c921a402-33ba-11e8-9f49-0a7f942386f8.jpg)

**Comment by Teabone3** [comment]
Is this only with the Easter DLC?

**Comment by Shahin3d** [comment]
Yes - I tested all the enemy characters - they all have this problem.

![snapshot47](https://user-images.githubusercontent.com/36647058/38140582-a7e39356-3449-11e8-9b88-543391ca54d1.jpg)
![snapshot49](https://user-images.githubusercontent.com/36647058/38140583-a8079710-3449-11e8-8eac-6f0d70c4680f.jpg)
![snapshot50](https://user-images.githubusercontent.com/36647058/38140584-a82e3ae6-3449-11e8-8730-14c71398669c.jpg)

**Comment by LeeBamberTGC** [comment]
This is the present expected behavior, enemies can traverse deep water (just like the player)., I have switched this to a FEATURE request, where a new flag in the enemy properties can be ticked to prevent them from entering the water below dry-weapon height.
